src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -221,15 +221,17 @@ def _create_global_aggregates(
 
 
 # Copied from transformers.models.t5.modeling_t5.T5LayerNorm with T5->LongT5
-class LongT5LayerNorm(nn.Module):
+class LongT5LayerNorm(nn.LayerNorm):
```

Comment:
This would create unused bias weights for every layernorm in T5 at the moment, I don't think we want this. @sgugger I'm actually not really in favor of adding this abstraction here to the `T5LayerNorm` class

Reply:
Only if the init calls the superclass. If it calls the super of the super class (nn.Module), there is nothing breaking. We just can't have the Trainer start depending on multiple modeling files because there are 100 flavors of LayerNorms.

An alternative would be to have a constant in `pt_utils` where each submodule that defines a specific LayerNorm adds their own, would that work better?

So here we would have after defining `LongT5LayerNorm`:
```
ALL_LAYERNORM_LAYERS.append(LongT5LayerNorm)
```

Reply:
@ADAning Could you amend your PR to use this solution maybe?

Reply:
Does `pt_utils` mean `transformers.utils` ?

> @ADAning Could you amend your PR to use this solution maybe?